net.sf.fuse_emulator.json

```diff
@@ -0,0 +1,115 @@
+{
+    "app-id": "net.sf.fuse_emulator",
+    "branch": "stable",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "fuse",
+    "rename-desktop-file": "fuse.desktop",
+    "rename-icon": "fuse",
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* Wayland access */
+        "--socket=wayland",
+        /* Sound */
+        "--socket=pulseaudio",
+        /* Joystick */
+        "--device=all",
+        /* Network access (for Spectranet) */
+        "--share=network",
+        /* Home directory (for loading and saving Spectrum software) */
+        "--filesystem=home",
+        /* Needed for dconf to work */
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1"
+        }
+    },
+    "cleanup": [
+        "/include",
+        "/lib/*.a",
+        "/lib/*.la",
+        "/lib/*.so",
+        "/lib/pkgconfig",
+        "/share/applications/mimeinfo.cache",
+        "/share/icons/hicolor/*/mimetypes",
+        "/share/icons/hicolor/icon-theme.cache",
+        "/share/man",
+        "/share/mime"
+    ],
+    "modules": [
+        {
+            /* Audiofile is used for loading Spectrum data from WAV files */
+            "name": "libaudiofile",
+            "config-opts": [
+                "--disable-docs",
+                "--disable-examples",
+                "--disable-flac",
+                "--disable-static"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://audiofile.68k.org/audiofile-0.3.6.tar.gz",
+                    "sha256": "cdc60df19ab08bfe55344395739bb08f50fc15c92da3962fac334d3bff116965"
+                },
+                {
+                    "type": "patch",
+                    "path": "audiofile-build.patch"
+                }
+            ],
+            "cleanup": [
+                "/bin"
+            ]
+        },
+        "shared-modules/SDL/SDL-1.2.15.json",
+        {
```

Comment:
Some leftovers from removal?

Reply:
What leftovers? I'm including the SDL json there :-?

Reply:
Oh, I see the "name": "libsdl" part is not necessary.

I was confused by the instructions here:

https://github.com/flathub/flathub/wiki/App-Requirements#shared-modules